langchain4j-mongodb-atlas/src/test/java/dev/langchain4j/store/embedding/mongodb/MongoDbEmbeddingStoreIT.java

```diff
@@ -0,0 +1,67 @@
+package dev.langchain4j.store.embedding.mongodb;
+
+import com.mongodb.client.MongoClient;
+import dev.langchain4j.data.segment.TextSegment;
+import dev.langchain4j.model.embedding.EmbeddingModel;
+import dev.langchain4j.store.embedding.EmbeddingStore;
+import dev.langchain4j.store.embedding.EmbeddingStoreIT;
+import dev.langchain4j.store.embedding.EmbeddingStoreWithFilteringIT;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.testcontainers.mongodb.MongoDBAtlasLocalContainer;
+
+import static dev.langchain4j.store.embedding.TestUtils.awaitUntilAsserted;
+import static dev.langchain4j.store.embedding.mongodb.MongoDbTestFixture.*;
+import static org.assertj.core.api.Assertions.assertThatNoException;
+
+public class MongoDbEmbeddingStoreIT extends EmbeddingStoreIT {
+
+    public static class ContainerIT extends MongoDbEmbeddingStoreIT {
+        static MongoDBAtlasLocalContainer mongodb = new MongoDBAtlasLocalContainer("mongodb/mongodb-atlas-local:7.0.9");
```

Comment:
What if we move this move this to the MongoDbTestFixture and call 
```java
MongoDbTestFixture.assertDoContainerTests();
mongodb.start();
```
in the fixture's constructor?

We can make MongoDbTestFixture a singleton and rename it to TestMongoDb

Reply:
Possibly this review comment is for an older commit - the [0af451e](https://github.com/langchain4j/langchain4j/pull/1932/commits/0af451ebcbf261064df9a3e8c1cb4fd8ed8d3f6f) commit above should accomplish this, no?